tools/gen_addon_readme.template

```diff
@@ -26,6 +26,15 @@
 {% for _ in badges %}|badge{{ loop.index }}| {% endfor %}
 
 {{ fragments.get('DESCRIPTION', '') }}
+{% if development_status == 'Alpha' -%}
+
+.. IMPORTANT::
+   This is an alpha version, the data model and design can change at any time without warning.
+   Only for development or testing purpose, do not use in production.
+   `More details on development status <https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/oca_module_lifecycle_development_status.rst>`_
```

Comment:
This must be changed to the canonical URL that this mentioned at the bottom of that document.

Reply:
But I think the canonical URL doesn't point to the exact anchor